### ISSUE TITLE
20180224

### ISSUE THREAD

**Issue by HurleyJames** [HurleyJames/IELTS_Writing#1]
As countries develop, more and more people buy and use their own cars. Do you think the advantage of this trend for individuals outweigh its disadvantages for the environment?

It is common for people to purchase and use their private cars frequently in daily life. I believe that the damage on the environment caused by this phenomenon overshadows the benefits for individuals. 
Admittedly, individuals can spend less time in traffic if they drive their own cars. If passengers miss a bus, they have to wait for several or even more than ten minutes for the next bus. However, after puschasing a car, people can drive to any place they want at any time, and they don’t need to waste time on waiting for public transportation. Nevertheless, the large number of private cars does more damage to the environment. 
To begin with, the use of private cars will lead to air pollution. It is acknowledged that carbon dioxide will be produced when cars burn fuels, and excessive carbon dioxide will aggravate global warming. If the temperature on the earth continues to grow, some coastal cities may disappear when icebergs melt and the sea level rises. Therefore, the air populltion caused by private cars is even likely to threaten the survival of human beings. 
In addition, water pollution arises from the use of cars. When people drive cars, more or less, cars will leak oil on the street. If the number of private cars soared, the amount of oil leaked on the street would also increase significantly. Rain and drainage will make leaded oil enter the water cycle, and then the whole water system is polluted. As a result, the water we drink every day may not be so clean, and such polluted water will also influence other specie’s life. 
To sum up, considering the air pollution and water pollution caused by the use of private cars, its disadvantage outweigh the advantage for individuals.
